frontend/src/app/tasks/hooks/useEditTaskForm.ts

```diff
@@ -51,5 +53,19 @@ export const useEditTaskForm = ({ task, tasks, closeForm }: UseEditTaskFormProps
     }
   }
 
-  return { formState, handleChange, handleSubmit, resetForm, handleProject }
+  useEffect(() => {
+    const handleKeyPress = (event: KeyboardEvent) => {
+      if (event.key === 's' && event.ctrlKey && formRef.current?.contains(document.activeElement)) {
+        event.preventDefault()
+        formRef.current?.requestSubmit()
+      }
+    }
+    document.addEventListener('keydown', handleKeyPress)
+
+    return () => {
+      document.removeEventListener('keydown', handleKeyPress)
```

Comment:
Why do we need these 2 keydown events?

Reply:
It's actually just one, the second part is removing the eventListener when the component unmounts

Reply:
Oh, got it now, sorry